### ISSUE TITLE
Rename 'Very' and 'Extremely Slow Sleep Healing' to Immunosuppression. Redescribe Slow Healer.

### ISSUE THREAD

**Issue by Pseudo-ku** [CleverRaven/Cataclysm-DDA#23381]
The addition of VSSH/ESSH is nice but it certainly doesn't roll off the tongue. Nor is it a simplified 'condition' in the grammatical sense - compare it with terms like 'Jittery' and 'Schizophrenic'

I believe [immunosuppression](https://en.wikipedia.org/wiki/Immunosuppression) is probably the most accurate description of VSSH/ESSH because it relates to the body not being able to heal as efficiently as normal. One-well known disease is HIV (human immunosuppressive/suppression(?) virus) but I wonder if that would step over the line a bit on trait descriptions. VSSH could be 'Immunosuppression' and ESSH could be 'Chronic Immunosuppression'

Also, what to do about Slow Healer? Slow healer's description doesn't explicitly state how much HP you lose, compared to VSSH/ESSH where you know what you're in for. Reminder, [Slow Healer is -0.0025](http://cddawiki.chezzo.com/cdda_wiki/index.php?title=Slow_Healer), which gameplay-wise is basically no difference.

Is it really wise/necessary to have 3 traits that affect the same thing - even if they don't stack?

Finally, it's worthwhile adding VSSH/ESSH to the wiki.

Thoughts?

**Comment by DracoGriffin** [comment]
I already tried with recommending hypopnea and apnea which are pretty close to what the effects they'd have if translated to CDDA, but it was turned down due to the supposedly can of worms that'd entail (having to develop C-PAP machines and the like).

However, I agree, the naming is horrible and really needs to be redone. Except I don't agree with Immunosuppression; that has nothing to do with sleeping or healing while sleeping.

**Comment by Pseudo-ku** [comment]
@DracoGriffin Fair enough. I agree that CPAP machines would be silly.

For me it's less about having an 'accurate' naming convention and more about having something simpler to remember/say.

**Comment by DracoGriffin** [comment]
Honestly, I could see CPAP machines being implemented very easily; similar to how welding goggles are checked. There would just be a check for a cpap machine adjacently if you had hypopnea/apnea when you went to sleep; if you didn't, you'd take the effect of the "mutation". You could even have makeshift versions with gas masks / oxygen tanks, that could reduce the healing malus by like 25%/50% or something.

And I'd much prefer selecting a name that reflects actual real-world stuff. I enjoy the sci-fi aspects, but it seems weird to just straight up ignore reality in other ways, but yet go into the realm of extradimensional aliens. :man_shrugging:

**Comment by Night-Pryanik** [comment]
>HIV (human immunosuppressive/suppression(?) virus)

That's human immunodeficiency virus.

**Comment by Pseudo-ku** [comment]
Thanks for clarifying. I wrote that while still waking up.

**Comment by moxian** [comment]
I'm not exactly sure how these mutations work (or how they work at all, really), but would it make sense to have them as one "progressive" mutation. I.e. you get "slow healing" that might then transform into "slow healing 2" and "slow healing 3"?
I don't think "slow healing" is a confusing name for base mutation. "For whatever reason, your body is slower than others at repairing damage". I buy that one easily.
And "<mutation name> <number>" also seems pretty intuitive to me. Yes, you need to exercise suspension of disbelief a little bit, but it's by no means worse than "cooking 5", "strength 8", "encumbrance 15" or any of the other arbitrary numbers the game has already.

Is "progressive mutations" already a thing in Cata? If no, can they be? (again, I've never encountered mutations during gameplay yet, so I have no idea how that works)

**Comment by Pseudo-ku** [comment]
@moxian Without spoiling too much you can get mutations that replace one another if they repeatedly mutate. For example, head bumps would be replaced with antennae.

Slow healing is not a confusing name, but the description is. Compared to VSSH/ESSH you don't actually know how much HP you're losing. If anything Slow Healing is the best name of the three.

A progressive system would be a great idea :)

**Comment by kevingranade** [comment]
I'm not against improving the name, but everything that has been suggested so far is * wildly* inaccurate.
Both apnea and immunosupression are conditions that have slow healing as a symptom, but not the most important symptom.  Apnea' s primary symptom is chronic fatigue, immunosupression's primary symptom is susceptibility to infection.

**Comment by Pseudo-ku** [comment]
You're right.

I think what the names need is a simplification. They sound and look starkly different to the theme of the game. Bring them into line with the Slow Healer trait.

**Comment by DracoGriffin** [comment]
> I'm not against improving the name, but everything that has been suggested so far is * wildly* inaccurate.
Both apnea and immunosupression are conditions that have slow healing as a symptom, but not the most important symptom. Apnea' s primary symptom is chronic fatigue, immunosupression's primary symptom is susceptibility to infection.

The primary symptom of [apnea and hypopnea](https://en.wikipedia.org/wiki/Sleep_deprivation#Sleep_apnea) is lack of oxygen, followed by interruption of NREM & REM sleep, both of which lead to regenerative issues -- that of which, chronic fatigue is one of them, because as you stated, the body is fatigued because [it did not regenerate properly](https://en.wikipedia.org/wiki/Sleep_deprivation#Physiological_effects), so the [homeostatic mechanisms of the body failed](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4651462/)
> Studies have shown that NREM sleep is important for turning off the norepinephrine, serotonin and histamine neurotransmitters, which in turn allows their receptors to “rest” and regain sensitivity [13]. This allows norepinephrine, serotonin and histamine to be more effective at naturally produced levels. During sleep, there are enzymes that repair brain cell damage caused by free radicals. In contrast, lack of sleep does not allow our brain to function normally because of the neurotransmitters, and neurons that are unable to rest or regenerate [13]. This becomes worse with people that are sleep deprived for days and their neurons start to degenerate because of constantly being actively at work. The neurons get worn out without having the time to regenerate and rest.

, [leaving you impaired upon arousal](https://en.wikipedia.org/wiki/Effects_of_sleep_deprivation_on_cognitive_performance).

By saying "chronic fatigue" as a symptom, you're avoiding the underlying cause and effect relationship, of which I hope to have clarified above.

**Comment by kevingranade** [comment]
@DracoGriffin You're trying way too hard to make this connection.  I HAVE apnea, I have tons of symptoms that are NOT slow healing, and I do not have a symptom remotely like slow healing.

If you have an IRL condition that matches, suggest it, but considering the fact that nothing suggested yet comes close, and it's already getting rather obscure, I very much doubt that there's a recognizable condition that has this effect as a primary symptom.

> And if the game design is lending itself to reality (to the nth degree) in other areas, why skimp on health and traits?

Because applying inaccurate labels based on real conditions is less realistic than applying accurate but arbitrary ones.

**Comment by DracoGriffin** [comment]
>@DracoGriffin You're trying way too hard to make this connection. I HAVE apnea, I have tons of symptoms that are NOT slow healing, and I do not have a symptom remotely like slow healing.
If you have an IRL condition that matches, suggest it, but considering the fact that nothing suggested yet comes close, and it's already getting rather obscure, I very much doubt that there's a recognizable condition that has this effect as a primary symptom.
Because applying inaccurate labels based on real conditions is less realistic than applying accurate but arbitrary ones.

I'm not trying to get the slow healing to be named to the apnea suggestions anymore, I dropped that in the last discussion. I am merely correcting the misinterpretation of the process of healing and how vital sleep is for the body to maintain homeostasis, of which healing is a byproduct of those mechanisms.

As far as I know, there is no condition that more directly correlates decreased homeostatic regeneration during sleep (other than some rare 1 in a million condition that would only be in a cited in a medical book or sleep specialist journals), other than what I have reported to you, as NREM and REM, as well as oxygenation, all play equally vital roles in maintaining health.

[Factors Affecting Wound Healing](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2903966/):
>Wound healing is a complex biological process that consists of hemostasis, inflammation, proliferation, and remodeling. Large numbers of cell types—including neutrophils, macrophages, lymphocytes, keratinocytes, fibroblasts, and endothelial cells—are involved in this process. Multiple factors can cause impaired wound healing by affecting one or more phases of the process and are categorized into local and systemic factors. **The influences of these factors are not mutually exclusive**. **Single or multiple factors may play a role in any one or more individual phases, contributing to the overall outcome of the healing process**.

>Stressors can lead to negative emotional states, such as anxiety and depression, which may in turn have an **impact on physiologic processes and/or behavioral patterns that influence health outcomes**. In addition to the direct influences of anxiety and depression on endocrine and immune function, stressed individuals are more likely to have unhealthy habits, which include **poor sleep patterns**, inadequate nutrition, less exercise, and a greater propensity for abuse of alcohol, cigarettes, and other drugs. **All of these factors may come into play in negatively modulating the healing response**. The effects of stress on wound healing are summarized in Fig. 1.

>Oxygenation
Oxygen is important for cell metabolism, especially energy production by means of ATP, and is **critical for nearly all wound-healing processes**. It prevents wounds from infection, induces angiogenesis, increases keratinocyte differentiation, migration, and re-epithelialization, enhances fibroblast proliferation and collagen synthesis, and promotes wound contraction (Bishop, 2008; Rodriguez et al., 2008). In addition, the level of superoxide production (a key factor for oxidative killing pathogens) by polymorphonuclear leukocytes is critically dependent on oxygen levels.
Due to vascular disruption and high oxygen consumption by metabolically active cells, the microenvironment of the early wound is depleted of oxygen and is quite hypoxic. Several systemic conditions, including advancing age and diabetes, can create impaired vascular flow, thus setting the stage for poor tissue oxygenation. In the context of healing, this overlay of poor perfusion creates a hypoxic wound. Chronic wounds are notably hypoxic; tissue oxygen tensions have been measured transcutaneously in chronic wounds from 5 to 20 mm Hg, in contrast to control tissue values of 30 to 50 mm Hg (Tandara and Mustoe, 2004).
In wounds where oxygenation is not restored, healing is impaired. **Temporary hypoxia after injury triggers wound healing, but prolonged or chronic hypoxia delays wound healing** (Bishop, 2008; Rodriguez et al., 2008). In acute wounds, hypoxia serves as a signal that stimulates many aspects of the wound-healing process. Hypoxia can induce cytokine and growth factor production from macrophages, keratinocytes, and fibroblasts. Cytokines that are produced in response to hypoxia include PDGF, TGF-β, VEGF, tumor necrosis factor-α (TNF-α), and endothelin-1, and are crucial promoters of cell proliferation, migration and chemotaxis, and angiogenesis in wound healing (Rodriguez et al., 2008).
In normally healing wounds, ROS such as hydrogen peroxide (H2O2) and superoxide (O2) are thought to act as cellular messengers to stimulate key processes associated with wound healing, including cell motility, cytokine action (including PDGF signal transduction), and angiogenesis. **Both hypoxia and hyperoxia increase ROS production, but an increased level of ROS transcends the beneficial effect and causes additional tissue damage** (Rodriguez et al., 2008).
In summary, **the proper oxygen level is crucial for optimum wound healing. Hypoxia stimulates wound healing such as the release of growth factors and angiogenesis, while oxygen is needed to sustain the healing process** (Bishop, 2008). One therapeutic option that can sometimes overcome the influence of tissue hypoxia is hyperbaric oxygen therapy (HBOT; Rodriguez et al., 2008). While HBOT can be an effective treatment for hypoxic wounds, its availability is limited.

Emphasis mine. And when can hypoxia occur? During apneic episodes of those suffering from sleep apnea or sleep hypopnea, or [any other pathological respiratory distress](https://en.wikipedia.org/wiki/Template:Respiratory_system_symptoms_and_signs).

**Comment by kevingranade** [comment]
Feel free to reopen if you have candidate names that don't have the issues outlined in the discussion.
